oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/IndexInfoServiceImpl.java

```diff
@@ -62,11 +63,21 @@ public IndexInfoServiceImpl(NodeStore nodeStore, IndexPathService indexPathServi
 
     @Override
     public Iterable<IndexInfo> getAllIndexInfo() {
+        HashSet<String> allIndexes = new HashSet<>();
+        indexPathService.getIndexPaths().forEach(allIndexes::add);
+        HashSet<String> activeIndexes = new HashSet<>();
+        if (indexPathService.getMountInfoProvider().hasNonDefaultMounts()) {
+            IndexName.filterReplacedIndexes(allIndexes, nodeStore.getRoot(), true).forEach(activeIndexes::add);
+        } else {
+            activeIndexes.addAll(allIndexes);
+        }
         return Iterables.filter(Iterables.transform(indexPathService.getIndexPaths(), new Function<String, IndexInfo>() {
             @Override
             public IndexInfo apply(String indexPath) {
                 try {
-                    return getInfo(indexPath);
+                    IndexInfo info = getInfo(indexPath);
+                    info.setActive(activeIndexes.contains(indexPath));
+                    return info;
```

Comment:
Is it possible for getInfo to return null? the method can return null in some conditions, could it happen in this particular context? If so, guard against that case when calling `setActive`.